Adjust references to usermod(1M) to be usermod(8) in the pcap.3pcap man page.

This patch should be sent upstream.

--- libpcap-1.7.4/pcap.3pcap.in.orig	2016-04-20 08:29:21.594024650 -0700
+++ libpcap-1.7.4/pcap.3pcap.in	2016-04-20 08:29:44.364454401 -0700
@@ -244,7 +244,7 @@
 that privilege to the user's
 .B defaultpriv
 key with the
-.B usermod (1M)
+.B usermod (8)
 command.
 .TP
 .B Under HP-UX with DLPI:
@@ -915,7 +915,7 @@
 1.0 APIs are available and use them only if they are.
 .SH SEE ALSO
 autoconf(1), tcpdump(1), tcpslice(1), pcap-filter(@MAN_MISC_INFO@), pfconfig(8),
-usermod(1M)
+usermod(8)
 .SH AUTHORS
 The original authors of libpcap are:
 .LP
